.github/workflows/cli.yaml

```diff
@@ -48,11 +45,20 @@ jobs:
           export KUBECONFIG=${HOME}/karmada/karmada-apiserver.config
           GO111MODULE=on go install github.com/onsi/ginkgo/v2/ginkgo
           ginkgo -v --race --trace -p  --focus="[BasicPropagation] propagation testing deployment propagation testing"  ./test/e2e/
-      - uses: chainguard-dev/actions/kind-diag@main
-        # Only upload logs on failure.
-        if: ${{ failure() }}
+      - name: export logs 
+        if: always()
+        run: |
+          export ARTIFACTS_PATH=${{ github.workspace }}/karmadactl-test-logs/${{ matrix.k8s }}/
+          mkdir -p $ARTIFACTS_PATH
+
+          mkdir -p $ARTIFACTS_PATH/{karmada-host,member1,member2}
+          kind export logs --name=karmada-host $ARTIFACTS_PATH/karmada-host
+          kind export logs --name=member1 $ARTIFACTS_PATH/member1
```

Comment:
This workflow is used to test the functionality of `karmadactl init`, is it necessary to collect logs of member clusters?

Reply:
Also running the test of `BasicPropagation`, this is the reason what i want to collect them.

Reply:
If it's because the test of `BasicPropagation`, would it be enough to just collect the logs of `karmada control plane`?
Take `run-e2e.sh` for example, it only collects the logs of `karmada host plane` and `pull-mode-cluster`.  And through investigations, they collect the logs of `pull-mode-cluster` just for `karmada-agent`. 
  
https://github.com/karmada-io/karmada/blob/4898c0fc55a13033fcf21fcc46a41318548c5713/hack/run-e2e.sh#L59-L71